index.ts

```diff
@@ -631,6 +633,22 @@ export const deserializeIlpRejection = (binary: Buffer): IlpRejection => {
 
   // Ignore remaining bytes for extensibility
 
+  if (code === 'F08') {
+    const dataReader = Reader.from(data)
+    const highBitsRcv = dataReader.readUInt32()
```

Comment:
`oer-utils@2.0.1` has proper methods to read UInt64s

Reply:
I tried updating the oer-utils dependency but that makes the tests break, so will have to move that to a follow-up ticket.